Gruntfile.js

```diff
@@ -111,9 +111,18 @@ module.exports = (grunt) => {
 
   async function generateManifests (dir, version) {
     if (development) {
+      const keyFile = process.env.KEY_FILE || 'key.json'
+      const pubKeyFile = process.env.PUB_KEY_FILE || 'key.pub.json'
+      if (fs.existsSync(keyFile)) {
+        grunt.log.writeln(chalk.underline(`Key file ${keyFile} exists, using that.`))
+      } else {
+        grunt.log.writeln(chalk.underline(`\nRunning 'chel keygen --pubout ${pubKeyFile} --out ${keyFile}'`))
+        const { stdout } = await execWithErrMsg(`chel keygen --pubout ${pubKeyFile} --out ${keyFile}`)
+        console.log(stdout)
```

Comment:
Isn't this creating a situation where we're signing contracts with any random key?

Don't we want to restrict signing to a specific key?

Or is this fine because it will make it easier for new volunteers to submit PRs ... in which case we can re-sign the contracts for them?

Reply:
> Isn't this creating a situation where we're signing contracts with any random key?

Correct.

> Don't we want to restrict signing to a specific key?

Also correct.

> Or is this fine because it will make it easier for new volunteers to submit PRs

Also correct.

> in which case we can re-sign the contracts for them?

And also correct.

Note:
- This is run only during development (so, the key used is of little significance)
- During deployment, a specific trusted key should be used
- And ideally there'll be some review involved, even if the signing itself is automated somehow. Volunteers shouldn't ever have to sign a contract for deployment.